src/pages/dashboardTab/components/tabController.js

```diff
@@ -40,6 +90,49 @@ const tabController = (classes) => {
 && state.dashboardTab.datatable
     ? state.dashboardTab.datatable : {}));
 
+  const tooltipContent = {
+    0: 'Click button to add selected files associated with selected cases(s)',
```

Comment:
During custodian try cases, samples, and files instead of 0,1,2

Reply:
0,1,2 are more generic as tab may not use for case, sample and files in the future.